content/en/real_user_monitoring/browser/collecting_browser_errors.md

```diff
@@ -140,6 +140,88 @@ try {
 {{% /tab %}}
 {{< /tabs >}}
 
+### Collect React rendering errors
+
+Error boundaries are React components that catch rendering errors anywhere in its child component tree and display a fallback UI. React error boundaries offer the capability to collect the component stack. Compared to error stack traces, React error boundaries provide more detailed information about the hierarchy of components involved. The additional information helps developers identify and fix errors.
+
+If your React application throws an error during rendering, React removes its UI from the screen by default. To modify this behavior, you can instrument the React router to monitor React rendering errors with the RUM Browser SDK `addError()` API.
+
+The collected rendering error stack contains the component stack, which is unminified like any other error stack traces once you've uploaded sourcemaps.
```

Comment:
People looking for a React integration are likely already familiar with Error boundaries. So IMO, there is no need to describe the behaviour of React Error boundaries. 
We could stay concise like for [React router integration](https://docs.datadoghq.com/real_user_monitoring/browser/modifying_data_and_context/?tab=npm#react-router-instrumentation) and only say:


```suggestion
You can instrument the React error boundaries to monitor React rendering errors using the RUM Browser SDK addError() API.
The collected rendering error stack contains the component stack, which is unminified like any other error stacktraces once you’ve uploaded sourcemaps.

To instrument React error boundaries for RUM monitoring, use the following:
```